app/src/Pages/AddContribution/AddContribution.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from "react";
```

Comment:
I think we should move this code into the app/src/Pages/Portal/Contributions/Contributions.js page (so combining the two pages).

Reply:
Correction, it looks the pages you are creating should represent the path `/contributions/add` so maybe in that page. I would create the page here:
`/app/src/Pages/Portal/Contributions/Add` (we are trying to group the portal pages similar to how you would in Next.js)

Reply:
👍